Script/PRONTO.py

```diff
@@ -749,13 +883,14 @@ def update_ppt_variant_summary_table(data_nrows,DNA_sampleID,RNA_sampleID,TMB_DR
 	return stable_text
 
 
-def remisse_mail_writer(remisse_file,ipd_no,ipd_consent,DNA_normal_sampleID,RNA_sampleID,extraction_hospital,ipd_material_id,TMB_DRUP,stable_text,sample_material,sample_type,sample_list,pipline):
+def remisse_mail_writer(remisse_file,ipd_no,ipd_consent,DNA_normal_sampleID,RNA_sampleID,extraction_hospital,ipd_material_id,str_TMB_DRUP,TMB_DRUP,stable_text,sample_material,sample_type,sample_list,pipline):
 	from docx import Document
```

Comment:
could these be imported at the top of the file as well?

Reply:
same here, maybe put it in a separate module and import on top of the file.

Reply:
An issue is created for it.